src/shared/lib/fetch/get.js

```diff
@@ -57,7 +58,7 @@ export const get = async (url, type, date = process.env.SCRAPE_DATE || datetime.
       tries++;
       if (tries > 1) {
         // sleep a moment before retrying
-        console.log(`  ⚠️  Retrying (${tries})...`);
+        log(`  ⚠️  Retrying (${tries})...`);
```

Comment:
Even though this is technically a warning, I say let it stay as `log` -- we don't want to see this come up unless it becomes an error (down below)